.github/workflows/accept.yml

```diff
@@ -0,0 +1,214 @@
+name: Fallback Install, Build & Package & Prebuilt Install (Merge)
+
+on: 
+  push: 
+    branches: 
+      # triggered by pull request approval triggers which is a merge to default branch
+      - master
+
+  workflow_dispatch:
+    inputs: 
+      target-test:
+        required: false
+        description: 'Run Tests on Target & Fallback Groups? (type: yes)'
+        default: false
+
+jobs:
+  # both build-group-unpublish and  build-group-publish "needs" this job
+  load-build-group:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout ${{ github.ref }}
+      uses: actions/checkout@v2
+
+      # build with the lowest versions of the OSes supported so the glibc/musl versions are the oldest/most compatible. 
+      # note: some of those images are no longer supported officially (https://hub.docker.com/_/node)
+    - name: Load build group data
+      id: set-matrix
+      run: .github/scripts/matrix-from-json.sh .github/config/build-group.json
+
+  build-group-unpublish:
+    runs-on: ubuntu-latest 
+    needs: load-build-group
+    strategy:
+      fail-fast: false
```

Comment:
Job who `needs` another job won't run regardless of the `fail-fast:` setting. Because if one in a matrix failed the job as whole failed. That being said, we can (and should) indeed `fail-fast: true` on this job (and some others). Good catch.

Reply:
Good to know about the fail-fast and matrix job failure status :) And agree doesn't seem to hurt for this job not to fail fast.

Reply:
Updating comments is ~~fun~~ hard work!